update/update.go

```diff
@@ -1,17 +1,87 @@
-package base
+// Package update defines the update service
+package update
 
 import (
+	"context"
 	"fmt"
 	"os/exec"
 	"path/filepath"
 
+	golog "github.com/ipfs/go-log"
 	"github.com/qri-io/dataset"
 	"github.com/qri-io/ioes"
 	"github.com/qri-io/iso8601"
-	"github.com/qri-io/qfs"
-	"github.com/qri-io/qri/cron"
+	"github.com/qri-io/qri/config"
+	"github.com/qri-io/qri/update/cron"
 )
 
+var log = golog.Logger("update")
+
+// Start starts the update service
+func Start(ctx context.Context, repoPath string, updateCfg *config.Update, daemonize bool) error {
+	if updateCfg == nil {
+		updateCfg = config.DefaultUpdate()
+	}
+
+	cli := cron.HTTPClient{Addr: updateCfg.Address}
```

Comment:
Can you name this variable `httpClient`? Seeing `cli` makes me think "command-line interface". I know we use this for rpc everywhere, but 1) at least that's a consistent pattern, 2) it's confusing to use the same name for a non-rpc client.